mods/tuxemon/maps/downstairs_test.tmx

```diff
@@ -85,5 +85,21 @@
    </properties>
   </object>
   <object id="28" name="Player Spawn" type="event" x="80" y="64" width="16" height="16"/>
+  <object id="29" name="create mom" type="event" x="16" y="48" width="16" height="16">
+   <properties>
+    <property name="act1" value="create_npc npc_mom,9,4,girl1,stand"/>
+    <property name="act2" value="npc_face npc_mom,left"/>
+    <property name="act3" value="npc_speed npc_mom, 0.2"/>
+    <property name="act4" value="npc_wander npc_mom"/>
+    <property name="cond1" value="not npc_exists npc_mom"/>
+    <property name="cond2" value="is player_facing down"/>
```

Comment:
Do we need this?
    <property name="cond2" value="is player_facing down"/>

Reply:
We shouldn't, but in my testing, npc_mom wouldn't spawn at all without it

Reply:
Are you sure? Can't reproduce.